src/index.spec.js

```diff
@@ -147,22 +148,19 @@ describe("provideState", () => {
 
     it("throws if an invalid state entry is assigned", () => {
       const { effects } = makeTestInstance({
-        initialState: () => ({
-          corge: "qux",
-        }),
+        initialState: () => ({}),
         effects: {
           foo() {
             return { qux: 3 };
           },
         },
       });
-      return effects.foo().catch(error => {
-        expect(error).toEqual(
-          new Error(
-            '"qux" is not a valid state entry. If you want to use it, initialize it in "intialState"'
-          )
-        );
-      });
+
+      expect(effects.foo()).resolves.toThrowError(
```

Comment:
Does not work correctly, try to trigger the error.

Reply:
I forgot to add return before expect :( It's fixed now
https://jestjs.io/docs/en/expect#rejects